packages/dd-trace/test/telemetry/logs/log-collector.spec.js

```diff
@@ -29,15 +28,15 @@ describe('telemetry log collector', () => {
 
     it('should store logs with same message but different stack', () => {
       const ddFrame =
-        `at T (${ddBasePath}packages/dd-trace/test/telemetry/logs/log-collector.spec.js:29:21)`
+        `at T (${ddBasePath}dd-trace/test/telemetry/logs/log-collector.spec.js:29:21)`
```

Comment:
It's worriying that the tests in this file never failed in the old state. It basically means that the content of this variable has no impact on the result of the test 🤷

Reply:
Just very briefly looking at it it seems like we could make the test stricter.